Source/miniwin/misc_msg.cpp

```diff
@@ -18,7 +18,6 @@
 #include "utils/log.hpp"
 #include "miniwin/miniwin.h"
 
-

```

Comment:
Looks like there are no changes to this file and just formatting.

Reply:
Yes, clang-format was not applied to it, so every time I save this file it makes those changes. Has clang-format no been applied to the whole project?